src/modules/pos/validator/components/ChangeCommission/Form/ChangeCommissionForm.test.js

```diff
@@ -57,6 +57,32 @@ describe('ChangeCommissionForm', () => {
       expect(button.closest('button').disabled).toBeTruthy();
     });
 
+    it('should not pass when commission increases more than 5%', async () => {
+      const { getByTestId, getByText } = renderWithRouterAndQueryClient(
+        ChangeCommissionForm,
+        props
+      );
+      const value = '96';
+      const button = getByText(buttonText);
+      const input = getByTestId('newCommission');
+      fireEvent.change(input, { target: { value } });
+      expect(button.closest('button').disabled).toBeTruthy();
+      expect(screen.getByText('You cannot increase commission more than 5%')).toBeTruthy();
+    });
+
+    it('should not pass when new commission have more than 2 decimal places', async () => {
+      const { getByTestId, getByText } = renderWithRouterAndQueryClient(
+        ChangeCommissionForm,
+        props
+      );
+      const value = '89.0101';
+      const button = getByText(buttonText);
+      const input = getByTestId('newCommission');
+      fireEvent.change(input, { target: { value } });
+      expect(screen.getByText('Input decimal places limited to 2')).toBeTruthy();
+      expect(button.closest('button').disabled).toBeTruthy();
+    });
+
```

Comment:
I think it would also be nice to assert agains negative commission value inputed @oskarleonard

Reply:
Added, although that part of the logic was not added in this branch